src/core/consent/hash.js

```diff
@@ -0,0 +1 @@
+export default object => JSON.stringify(object);
```

Comment:
This is just a placeholder; it will need to change to a real hashing function. Preferably where the order of the object keys doesn't change the hash output, and where the resulting string is small (maybe base64 encoded).